FLT/DedekindDomain/FiniteAdeleRing/BaseChange.lean

```diff
@@ -219,22 +217,43 @@ noncomputable def adicCompletionComapAlgHom
     subst hvw
     simp only [RingHom.toMonoidHom_eq_coe, OneHom.toFun_eq_coe, MonoidHom.toOneHom_coe,
       MonoidHom.coe_coe]
-    have : (adicCompletionComapRingHom A K _ w rfl) (r : adicCompletion K (comap A w))  =
+    have : (adicCompletionComapRingHom A K _ w rfl) (algebraMap _ _ r)  =
         (algebraMap L (adicCompletion L w)) (algebraMap K L r) := by
       letI : UniformSpace L := w.adicValued.toUniformSpace
       letI : UniformSpace K := (comap A w).adicValued.toUniformSpace
       rw [adicCompletionComapRingHom, UniformSpace.Completion.mapRingHom]
-      rw [show (r : adicCompletion K (comap A w)) = @UniformSpace.Completion.coe' K this r from rfl]
       apply UniformSpace.Completion.extensionHom_coe
     rw [this, ← IsScalarTower.algebraMap_apply K L]
-  cont := sorry -- #235
+  cont :=
+    letI : UniformSpace K := v.adicValued.toUniformSpace;
+    letI : UniformSpace L := w.adicValued.toUniformSpace;
+    UniformSpace.Completion.continuous_extension
+
+omit [IsIntegralClosure B A L] [FiniteDimensional K L] [Algebra.IsSeparable K L] in
+lemma adicCompletionComapAlgHom_coe
+    (v : HeightOneSpectrum A) (w : HeightOneSpectrum B) (hvw : v = comap A w) (x : K) :
+    adicCompletionComapAlgHom A K L B v w hvw x = algebraMap K L x :=
+  (adicCompletionComapAlgHom A K L B v w hvw).commutes _
 
 -- this name is surely wrong
+open WithZeroTopology in
 lemma v_adicCompletionComapAlgHom
```

Comment:
This is the right name according to the naming convention.
We could probably rename `adicCompletionComapAlgHom` to `adicCompletionComap`.

Reply:
Your statement does not typecheck because $w$ is a valuation on $\hat{L}$ while $v$ is a valuation on $\hat{K}$.
So the statement is $w(f(x)) = v(x)^e$ where $f : \hat{K} \to \hat{L}$ is the map you called `adicCompletionComap`

Reply:
Fair enough!